### PR TITLE
Fix ruridit in luv AL

### PR DESCRIPTION
**Pull request by chochem**
Fixes https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/12729.

more work is needed to incorporate other BW replacements and then remove the replacement code in BW

### CODE REVIEW

File: src/main/java/gregtech/loaders/postload/recipes/AssemblyLineRecipes.java [hunk 1]
```diff
@@ -26,19 +29,140 @@ public void run() {
...
                 : FluidRegistry.getFluid("molten.solderingalloy");
 
+        if (isBartWorksLoaded) {
+            // Luv Motor, sensor, emitter, field generator
+            GT_Values.RA.addAssemblylineRecipe(
+                    ItemList.Electric_Motor_IV.get(1, new Object() {}),
+                    144000,
+                    new ItemStack[] { GT_OreDictUnificator.get(OrePrefixes.stick, Materials.SamariumMagnetic, 1L),
+                            GT_OreDictUnificator.get(OrePrefixes.stickLong, Materials.HSSS, 2L),
+                            getModItem("bartworks", "gt.bwMetaGeneratedwireFine", 64L, 90),
```
Comment (Connor-Colenso) [inline src/main/java/gregtech/loaders/postload/recipes/AssemblyLineRecipes.java:39]: Using wireFineRurdit with the GT Unifier is much easier to understand than getModItem with a random metadata.

File: src/main/java/gregtech/loaders/postload/recipes/AssemblyLineRecipes.java [hunk 2]
```diff
@@ -26,19 +29,140 @@ public void run() {
...
+                    ItemList.Electric_Motor_LuV.get(1),
+                    600,
+                    (int) TierEU.RECIPE_IV);
+
+            GT_Values.RA.addAssemblylineRecipe(
+                    ItemList.Emitter_IV.get(1, new Object() {}),
+                    144000,
+                    new Object[] { GT_OreDictUnificator.get(OrePrefixes.frameGt, Materials.HSSS, 1L),
+                            ItemList.Electric_Motor_LuV.get(1, new Object() {}),
+                            getModItem("bartworks", "gt.bwMetaGeneratedstick", 8L, 90),
```
Comment (Connor-Colenso) [inline src/main/java/gregtech/loaders/postload/recipes/AssemblyLineRecipes.java:52]: Ditto

File: src/main/java/gregtech/loaders/postload/recipes/AssemblyLineRecipes.java [hunk 3]
```diff
@@ -26,19 +29,140 @@ public void run() {
...
+                    ItemList.Emitter_LuV.get(1),
+                    600,
+                    (int) TierEU.RECIPE_IV);
+
+            GT_Values.RA.addAssemblylineRecipe(
+                    ItemList.Sensor_IV.get(1, new Object() {}),
+                    144000,
+                    new Object[] { GT_OreDictUnificator.get(OrePrefixes.frameGt, Materials.HSSS, 1L),
+                            ItemList.Electric_Motor_LuV.get(1, new Object() {}),
+                            getModItem("bartworks", "gt.bwMetaGeneratedplate", 8L, 90),
```
Comment (Connor-Colenso) [inline src/main/java/gregtech/loaders/postload/recipes/AssemblyLineRecipes.java:69]: Ditto

File: src/main/java/gregtech/loaders/postload/recipes/AssemblyLineRecipes.java [hunk 4]
```diff
@@ -26,19 +29,140 @@ public void run() {
...
+                    new FluidStack[] { new FluidStack(solderIndalloy, 144), Materials.Lubricant.getFluid(250) },
+                    ItemList.Electric_Motor_LuV.get(1),
+                    600,
+                    (int) TierEU.RECIPE_IV);
+
+            GT_Values.RA.addAssemblylineRecipe(
+                    ItemList.Emitter_IV.get(1, new Object() {}),
+                    144000,
+                    new Object[] { GT_OreDictUnificator.get(OrePrefixes.frameGt, Materials.HSSS, 1L),
+                            ItemList.Electric_Motor_LuV.get(1, new Object() {}),
```
Comment (Connor-Colenso) [inline src/main/java/gregtech/loaders/postload/recipes/AssemblyLineRecipes.java:117]: Why ", new Object() {}"?

File: src/main/java/gregtech/loaders/postload/recipes/AssemblyLineRecipes.java [hunk 5]
```diff
@@ -26,19 +29,140 @@ public void run() {
...
+                            GT_OreDictUnificator.get(OrePrefixes.stick, Materials.Osmiridium, 8L),
+                            ItemList.QuantumStar.get(1, new Object() {}),
```
Comment (Connor-Colenso) [inline src/main/java/gregtech/loaders/postload/recipes/AssemblyLineRecipes.java:119]: Ditto v2

File: src/main/java/gregtech/loaders/postload/recipes/AssemblyLineRecipes.java [hunk 6]
```diff
@@ -26,19 +29,140 @@ public void run() {
...
+                    new FluidStack[] { new FluidStack(solderIndalloy, 576) },
+                    ItemList.Emitter_LuV.get(1),
+                    600,
+                    (int) TierEU.RECIPE_IV);
+
+            GT_Values.RA.addAssemblylineRecipe(
+                    ItemList.Sensor_IV.get(1, new Object() {}),
+                    144000,
+                    new Object[] { GT_OreDictUnificator.get(OrePrefixes.frameGt, Materials.HSSS, 1L),
+                            ItemList.Electric_Motor_LuV.get(1, new Object() {}),
```
Comment (Connor-Colenso) [inline src/main/java/gregtech/loaders/postload/recipes/AssemblyLineRecipes.java:134]: Ditto v2

File: src/main/java/gregtech/loaders/postload/recipes/AssemblyLineRecipes.java [hunk 7]
```diff
@@ -26,19 +29,140 @@ public void run() {
...
+                    ItemList.Sensor_LuV.get(1),
+                    600,
+                    (int) TierEU.RECIPE_IV);
+
+            GT_Values.RA.addAssemblylineRecipe(
+                    ItemList.Field_Generator_IV.get(1, new Object() {}),
+                    144000,
+                    new Object[] { GT_OreDictUnificator.get(OrePrefixes.frameGt, Materials.HSSS, 1L),
+                            GT_OreDictUnificator.get(OrePrefixes.plate, Materials.HSSS, 6L),
+                            ItemList.QuantumStar.get(2, new Object() {}), ItemList.Emitter_LuV.get(4, new Object() {}),
```
Comment (Connor-Colenso) [inline src/main/java/gregtech/loaders/postload/recipes/AssemblyLineRecipes.java:152]: Ditto v2

File: src/main/java/gregtech/loaders/postload/recipes/AssemblyLineRecipes.java [hunk 8]
```diff
@@ -26,19 +29,140 @@ public void run() {
                 ? FluidRegistry.getFluid("molten.indalloy140")
                 : FluidRegistry.getFluid("molten.solderingalloy");
 
+        if (isBartWorksLoaded) {
+            // Luv Motor, sensor, emitter, field generator
+            GT_Values.RA.addAssemblylineRecipe(
+                    ItemList.Electric_Motor_IV.get(1, new Object() {}),
```
Comment (Connor-Colenso) [inline src/main/java/gregtech/loaders/postload/recipes/AssemblyLineRecipes.java:35]: Ditto v2

File: src/main/java/gregtech/loaders/postload/recipes/AssemblyLineRecipes.java [hunk 9]
```diff
@@ -26,19 +29,140 @@ public void run() {
...
+                            getModItem("bartworks", "gt.bwMetaGeneratedwireFine", 64L, 90),
+                            getModItem("bartworks", "gt.bwMetaGeneratedwireFine", 64L, 90),
+                            GT_OreDictUnificator.get(OrePrefixes.cableGt01, Materials.YttriumBariumCuprate, 2L) },
+                    new FluidStack[] { new FluidStack(solderIndalloy, 144), Materials.Lubricant.getFluid(250) },
+                    ItemList.Electric_Motor_LuV.get(1),
+                    600,
+                    (int) TierEU.RECIPE_IV);
+
+            GT_Values.RA.addAssemblylineRecipe(
+                    ItemList.Emitter_IV.get(1, new Object() {}),
```
Comment (Connor-Colenso) [inline src/main/java/gregtech/loaders/postload/recipes/AssemblyLineRecipes.java:48]: Ditto v2

File: src/main/java/gregtech/loaders/postload/recipes/AssemblyLineRecipes.java [hunk 10]
```diff
@@ -26,19 +29,140 @@ public void run() {
...
+                    144000,
+                    new Object[] { GT_OreDictUnificator.get(OrePrefixes.frameGt, Materials.HSSS, 1L),
+                            ItemList.Electric_Motor_LuV.get(1, new Object() {}),
```
Comment (Connor-Colenso) [inline src/main/java/gregtech/loaders/postload/recipes/AssemblyLineRecipes.java:51]: Ditto v2

File: src/main/java/gregtech/loaders/postload/recipes/AssemblyLineRecipes.java [hunk 11]
```diff
@@ -26,19 +29,140 @@ public void run() {
...
+                    new FluidStack[] { new FluidStack(solderIndalloy, 576) },
+                    ItemList.Emitter_LuV.get(1),
+                    600,
+                    (int) TierEU.RECIPE_IV);
+
+            GT_Values.RA.addAssemblylineRecipe(
+                    ItemList.Sensor_IV.get(1, new Object() {}),
+                    144000,
+                    new Object[] { GT_OreDictUnificator.get(OrePrefixes.frameGt, Materials.HSSS, 1L),
+                            ItemList.Electric_Motor_LuV.get(1, new Object() {}),
```
Comment (Connor-Colenso) [inline src/main/java/gregtech/loaders/postload/recipes/AssemblyLineRecipes.java:68]: Ditto v2

**Review by Connor-Colenso — CHANGES_REQUESTED** [review]
Etc etc, too many repeats

**Review by Dream-Master — CHANGES_REQUESTED** [review]
@Connor-Colenso @chochem please lets fix that inside BW. This is a huge mess when we need to find code.

**Comment by chochem** [comment]
can be done more thoroughly after stable

**Pull request closed without merging**